rootfs/api/models.py

```diff
@@ -311,11 +311,15 @@ def scale(self, user, structure):  # noqa
     def _scale_containers(self, scale_types, to_remove):
         release = self.release_set.latest()
         for scale_type in scale_types:
-            image = release.image
+            if not release.build.dockerfile and not release.build.sha:
```

Comment:
I feel we should make a function for this that better describes the intention of this code, since it's used so often